src/app/docs/card/card-docs.component.html

```diff
@@ -0,0 +1,22 @@
+<sky-docs-demo-page
+  moduleName="SkyCardModule"
+  moduleSourceCodePath="src/app/public/modules/card/"
+  pageTitle="Card"
+>
+  <sky-docs-demo-page-summary>
+    The card module creates a small container to highlight important information. 
+  </sky-docs-demo-page-summary>
+
+  <sky-docs-design-guidelines>
+    <sky-docs-demo-page-section
+      heading="UX guidelines"
```

Comment:
I'd change this to `Usage` or `Behavior and states`. Those are both standard sections in the phase 2 docs, and I think either lines up with the content here. (I was initially thinking `Content` because of the `Button text` subheader in the existing docs, but reading the content, I don't understand why we had that label. The guidance seems centered around general use, not the text in the cards.)